specification/dartLangSpec.tex

```diff
@@ -15980,9 +15955,9 @@ \subsection{Lexical Lookup}
 % Has chosen instance member, wants getter/setter, has only the opposite.
 When $\ell$ has access to \THIS{}
 (\ref{classes})
-and is located in a class \DefineSymbol{C},
+and is located in a class or mixin \DefineSymbol{A},
 it is a compile-time error if $D$ is an instance member,
```

Comment:
What is D here?  As best I can tell, at this point in the text, we haven't said anything about finding D or what it is.  Did this end up out of order in a refactor?

Reply:
`$D$` is defined before the commentary. It could be helpful to have the generated PDF available: It shows symbols in the right margin at the point where they are introduced (in the parts of the spec where I've done that, including this section).